plugins/katello/3.5/release_notes/release_notes.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: plugins/katello/documentation
+title: Release Notes
+version: 3.5
+---
+
+# Katello 3.5 (Schwarzbier) Release Notes
+
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.5/CHANGELOG.md).
+
+## Features
+
+#### Additional Supported Database Deployment Options
+
+[description]
+
+#### Composite Content View Versions
+
+[description]
```

Comment:
@jcpunk could you please provide a brief description of this feature?

Reply:
Can do, would you prefer I just dump them inline here or add a pull request?

Reply:
Inline is fine.

Reply:
Works for me :)

---

Content view and composite content view versions along with their publication dates are now provided to the clients as a structured fact.


---

If you want a sample of why this might be useful:   These facts are suitable for use in `/etc/motd` to provide end user feedback on the content accessible on a given host.